tests/UIButtons.js

```diff
@@ -0,0 +1,173 @@
+import Menu from "../page-objects/components/Menu";
+import HomePage from "../page-objects/pages/Homepage";
+
+describe("Checking the appearance of buttons", () => {
+  beforeEach(async () => {
+    HomePage.openHomePage();
+    Menu.waitForExistHeader();
+    Menu.clickOnApproveButton();
+    Menu.clickOnApproveButtonFirefox();
+    await browser.pause(5000);
+  });
+
+  it("Checking the appearance of the menu buttons", async () => {
```

Comment:
Maybe you should specify that you are referring to the menu buttons from the header